src/components/component.ts

```diff
@@ -234,6 +235,9 @@ export class Component {
    * Renders the Component without waiting for the next frame.
    */
   public renderImmediately() {
+    if (this._clipPathEnabled) {
+      this._updateClipPath();
+    }
```

Comment:
Usually just doing `clip-path="url(#clipPathID)"` (a relative path) would work ... but if there is a [`<base>` tag](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/base) on the page, it changes the base URL used for all relative paths ... causing the relative reference to the `clipPath` to get lost. The only way to work under a `<base>` tag is to specify an absolute path, as was done in https://github.com/palantir/plottable/pull/865.